hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -808,6 +810,26 @@ static void loginUserFromSubject(Subject subject) throws IOException {
       }
       loginUser = proxyUser == null ? realUser : createProxyUser(proxyUser, realUser);
 
+      String tokenFileLocation = System.getProperty(HADOOP_TOKEN_FILES);
```

Comment:
HADOOP_TOKEN_FILES can be set on the core-site.xml. However, if the user wants to set the new value using system parameter `-Dhadoop.token.files`, the parameter has high priority